modules/gdnative/nativescript/nativescript.h

```diff
@@ -152,6 +152,13 @@ class NativeScript : public Script {
 	virtual void set_source_code(const String &p_code) override;
 	virtual Error reload(bool p_keep_state = false) override;
 
+#ifdef TOOLS_ENABLED
+	virtual const Vector<DocData::ClassDoc> &get_documentation() const override {
```

Comment:
You should make the dependency explicit by adding the `#include` (include what you use).
But the problem of the `editor/doc_data.h` dependency in non-editor code needs to be solved first.

Reply:
done!